modules/flight_input/flight_input_worker.py

```diff
@@ -0,0 +1,31 @@
+import time
+
+from utilities.workers import queue_proxy_wrapper
+from utilities.workers import worker_controller
+from . import flight_input
+
+def flight_input_worker(address: str,
+                        period: float,
```

Comment:
Any particular reason for a period? It's in the video input worker to deliberately slow the detect target runs.